pandas/tests/reshape/concat/test_concat.py

```diff
@@ -1845,124 +1845,6 @@ def test_concat_preserves_extension_int64_dtype():
     tm.assert_frame_equal(result, expected)
 
```

Comment:
There is typo in commit message, it should be "split class TestSeriesConcat to file test_series.py"